### DISCUSSION TITLE
Daily Note Custom Command

### DISCUSSION THREAD

**Discussion by tavlean** [notaapp/talk#193] [Questions] [answered]
I'm very new to custom commands.

Can someone please help me set up a custom command for a daily note?
A command to create (or Open) a note with today's date every time the app is loaded.

I read the docs but couldn't understand how to do this.

**Comment by astoilkov** [comment]
We've already implemented Daily Notes and the 0.36 release will include them.

Meanwhile, you can create a custom command. I suppose that the missing piece is that you should open the newly created file with the `nota` CLI which can be installed via the `Install global "nota" CLI...` command in the command palette.

**Reply by bellebethcooper** [reply]
Is the daily notes feature in 0.36, or did it not make it? Unless it's part of the new templates feature, I didn't see it there.

**Reply by astoilkov** [reply]
It didn't make it. We have a little more work on it. The next release will include it.

**Reply by bellebethcooper** [reply]
Okay cool. Looking forward to trying it out

**Comment by tavlean** [comment]
I have a hard time understanding how this will work. A video tutorial or a step-by-step doc for this would be great.

For now, I'll wait for the 0.36 for daily notes. But I am looking forward to learning how to execute custom commands and extend the app's functionality.

**Reply by astoilkov** [reply]
We are working on improving our documentation on extensibility. We've planned a lot more for extensibility so we hope this makes it easier for everybody to make extensions, so stay tuned.

**Comment by astoilkov** [comment, marked as answer]
Daily notes are now available in the latest 0.37 release. The next release will include improvements and it will no longer be an experiment.
